lib/packwerk/inflector.rb

```diff
@@ -8,20 +8,37 @@
 module Packwerk
   class Inflector
     class << self
+      extend T::Sig
+
       def default
```

Comment:
FYI - The instance of `Packwerk::Inflector` returned by `default` is used internally to inflect words output by Packwerk (e.g. "5 violations found"). It is a different use case and irrelevant to this bug.

Reply:
I am not quite sure why we used `Packwerk::Inflector` for general UI inflections either. To add on to that, using `Packwerk::Inflector` in two different use cases also confused me when I was digging around. We should probably just give it one concern. 

I will create another PR for this after.

Reply:
Might be worth digging for the PR that introduced `Packwerk:Inflector.default` to see if a convincing reason was given.

Reply:
It seems like `Packwerk:Inflector.default` was used for `Packwerk:Inflector#pluralize`. `ActiveSupport::Inflector`'s [version of pluralize does not take in an argument](https://github.com/Shopify/packwerk-old/pull/150#discussion_r422123662), so we created the pluralization logic in `Packwerk:Inflector#pluralize`.